src/core/features.zig

```diff
@@ -57,23 +57,21 @@ pub const map: std.EnumArray(Feature, Info) = map: {
 ///
 /// [agave] https://github.com/anza-xyz/agave/blob/8db563d3bba4d03edf0eb2737fba87f394c32b64/sdk/feature-set/src/lib.rs#L1188
 pub const Set = struct {
-    array: std.EnumArray(Feature, Slot),
+    array: std.EnumArray(Feature, ?Slot),
```

Comment:
I was thinking about doing the same thing. It uses more memory but I think it's worth it for clarity/safety.

Reply:
wow interesting. you could pick a random number instead as the terminator. but still I prefer nullable